.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "python.pythonPath": "/Users/michaelzhang/.local/share/virtualenvs/hyposoft-Lu6dRYM6/bin/python"
+}
```

Comment:
could you send a push up that removes this file and folder from git?
likely you'll want to do
`git rm -rf .vscode/`
then add the following line to your .gitignore
`.vscode/`